### PR TITLE
Polish language dictionary

### PR DESCRIPTION
**Pull request by strugman**
*Issue #, if available:*
None

*Description of changes:*
Added Polish language dictionary

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/ui/src/i18n/dictionaries/index.ts [hunk 1]
```diff
@@ -5,3 +5,4 @@ export { frDict } from './fr';
 export { itDict } from './it';
 export { jaDict } from './ja';
 export { zhDict } from './zh';
+export { plDict } from './pl';
```
Comment (hvergara) [inline packages/ui/src/i18n/dictionaries/index.ts:8]: exports seems to be ordered alphabetically. Can you put it between `jaDict` and `zhDict` instead, please?

Comment (strugman) [reply packages/ui/src/i18n/dictionaries/index.ts:7]: Done :-)

**Comment by strugman** [comment]
Im closing this PR since the changes require another PR

**Comment by ericclemmons** [comment]
@strugman Is it fair to re-open it if we resolve the conflicts?

This translation & effort is very welcome! 🙏

**Comment by strugman** [comment]
Hello
I'm not sure I understand - are you suggesting to reopen this PR and try to resolve the conflicts? I have to admit it was the first time I forked a repo and contributed. For this reason I've messed up a bit, because when I found significant conflicts, I didn't know how to resolve them across 2 repos.
So I decided to close this PR and open another one (it was easier to bring my forked repo up to date with the new one and then apply the changes again and push): https://github.com/aws-amplify/amplify-ui/pull/828.
Would that be OK or do we want to continue in this one?

**Comment by wlee221** [comment]
The new PR looks good! Thanks @strugman.

**Pull request closed without merging**